neutron/neutron.go

```diff
@@ -30,12 +30,13 @@ const (
 
 // NetworkV2 contains details about a labeled network
 type NetworkV2 struct {
-	Id                string   `json:"id"` // UUID of the resource
-	Name              string   // User-provided name for the network range
-	SubnetIds         []string `json:"subnets"`         // an array of subnet UUIDs
-	External          bool     `json:"router:external"` // is this network connected to an external router
-	AvailabilityZones []string `json:"availability_zones"`
-	TenantId          string   `json:"tenant_id"`
+	Id                  string   `json:"id"` // UUID of the resource
+	Name                string   // User-provided name for the network range
+	SubnetIds           []string `json:"subnets"`         // an array of subnet UUIDs
+	External            bool     `json:"router:external"` // is this network connected to an external router
+	AvailabilityZones   []string `json:"availability_zones"`
+	TenantId            string   `json:"tenant_id"`
+	PortSecurityEnabled *bool    `json:"port_security_enabled"`
```

Comment:
Why a `*bool` here rather than a `bool`? (I guess you need for it to be nil-able?) Should `External` be `*bool` too?